agent.go

```diff
@@ -255,18 +262,18 @@ func (a Agent) Query(canisterID principal.Principal, methodName string, args []a
 }
 
 // RequestStatus returns the status of the request with the given ID.
-func (a Agent) RequestStatus(canisterID principal.Principal, requestID RequestID) ([]byte, hashtree.Node, error) {
-	a.logger.Printf("[AGENT] REQUEST STATUS %s", requestID)
+func (a Agent) RequestStatus(ecID principal.Principal, requestID RequestID) ([]byte, hashtree.Node, error) {
```

Comment:
NIT: `ecID` to `effectiveCanisterID`.

Reply:
Sure! Wanted to keep it consistent with the value returned by `effectiveCanisterID()` (and avoid confusion/shadowing between the value and the function)

Reply:
This is a good point, lets keep it as `ecid`!